nixos/modules/programs/sway.nix

```diff
@@ -117,7 +125,9 @@ in {
       }
     ];
     environment = {
-      systemPackages = [ swayPackage ] ++ cfg.extraPackages;
+      systemPackages = [ swayPackage ]
+        ++ (if cfg.useXwayland then [config.services.xwayland.package] else [])
```

Comment:
Isn't the xwayland module already installing the package?

Reply:
whoops, sorry about that
Fixed!